src/ui/public/filter_bar/lib/map_script.js

```diff
@@ -1,23 +1,15 @@
-export function FilterBarLibMapScriptProvider(Promise, courier) {
+export function FilterBarLibMapScriptProvider(Promise) {
   return function (filter) {
     if (filter.script) {
-      return courier
-      .indexPatterns
-      .get(filter.meta.index).then(function (indexPattern) {
-        const type = 'scripted';
-        const key = filter.meta.field;
-        const field = indexPattern.fields.byName[key];
+      const type = 'scripted';
+      const key = filter.meta.field;
 
-        let value;
-        if (filter.meta.formattedValue) {
-          value = filter.meta.formattedValue;
-        } else {
-          value = filter.script.script.params.value;
-          value = field.format.convert(value);
```

Comment:
This `else` was pointless because [map_phrase handles](https://github.com/Bargs/kibana/blob/e4c371af2583dd4134c026a14e2bd8c6431d2238/src/ui/public/filter_bar/lib/map_phrase.js#L33-L36) script filters that contain a `params.value` property (and appears before map_script in the mapping chain). I'm actually wondering if this map_script.js file is *ever* used. I dunno where `meta.formattedValue` would come from. Maybe @lukasolson has an idea?

Reply:
Hmm, I think honestly when I added filter editors I updated the `mapRange` and `mapPhrase` functions to properly map scripted fields, so I think you're probably right... This might not ever get called.

Reply:
@spalger it looks like the `formattedValue` stuff was added in [this PR](https://github.com/elastic/kibana/pull/4677). Any insights here? Not sure if it's safe to delete or if `mapRange` and `mapPhrase` should also be checking for a `meta.formattedValue` prop?

Reply:
Sorry, I really have no idea 😬